app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -74,7 +69,7 @@ def valid_auth_tokens
 
     def analytics_properties
       {
-        rendered_event_count: security_event_tokens.keys.count,
+        rendered_event_count: 1, # FIXME
```

Comment:
We've, for now, hijacked `security_event_tokens` to just return a string. This existing code was blowing up so I just hardcoded 1 while we work.

We need to find away to populate this, though, because it's valuable data.

Reply:
you could set `@rendered_event_count` in the part that reads events from the client class?